### PR TITLE
Fixed all bugs

### PR DESCRIPTION
**Pull request by TrevorSpear**
Fixed all bugs

### CODE REVIEW

File: src/main/java/edu/oregonstate/cs361/battleship/Main.java [hunk 1]
```diff
@@ -250,8 +250,8 @@ public static BattleshipModelUpdated placeShipUpdatedEasy(BattleshipModelUpdated
             Ship[] CArray = model.resetArrayUpdated(model, false);
             int computerLength = model.getShipByID(CArray[id].getName()).getLength() - 1;
 
-            model.getShipByID(CArray[id].getName()).setStart(id, id);
-            model.getShipByID(CArray[id].getName()).setEnd(id + computerLength, id);
+            model.getShipByID(CArray[id].getName()).setStart(id + 1, id + 1);
```
Comment (orrelln) [inline src/main/java/edu/oregonstate/cs361/battleship/Main.java:253]: Can you explain what issue this was causing?

**Comment by orrelln** [comment]
I mean actually its pretty clean and there isn't much needed to be commented on.

File: src/main/java/edu/oregonstate/cs361/battleship/Main.java (hunk 1 above)
Comment (TrevorSpear) [reply src/main/java/edu/oregonstate/cs361/battleship/Main.java:253]: Otherwise the first ship (aircraft carrier) starts at 0,0 to 0,4 (aka off the board)

Comment (TrevorSpear) [reply src/main/java/edu/oregonstate/cs361/battleship/Main.java:253]: with the +1 now aircraft carrier start at 1,1 to 1, 5

**Pull request merged**